chain/x/zoracle/internal/keeper/request.go

```diff
@@ -70,6 +115,15 @@ func (k Keeper) CheckRequestExists(ctx sdk.Context, id int64) bool {
 	return store.Has(types.RequestStoreKey(id))
 }
 
+// HasToPutInPendingList return boolean that request must be put in pending list or not.
```

Comment:
// returns whether the request can be moved to the pending resolve list.

Let's name this function `ShouldResolve`

Reply:
// ShouldBecomePendingResolve checks and returns whether the given request should be moved to the 
// pending resolve list, which will be resolved during the EndBlock call. The move will happen exactly when
// the request receives sufficient raw reports from the validators.

Ok. Let's call this `ShouldBecomePendingResolve`. We will call the list `PendingResolveList`